universal-application-tool-0.0.1/app/services/applicant/ApplicantService.java

```diff
@@ -44,4 +46,6 @@
       long applicantId, long programId);
 
   String applicantName(Application application);
+
+  CompletionStage<ImmutableList<ProgramDefinition>> relevantPrograms(long applicantId);
```

Comment:
(blocking) service interface methods should have good docstrings!

Reply:
Done!